core/service/src/config.rs

```diff
@@ -120,11 +122,17 @@ impl<C, G, E> Configuration<C, G, E> where
 {
 	/// Create default config for given chain spec.
 	pub fn default_with_spec(chain_spec: ChainSpec<G, E>) -> Self {
+		Self::default_with_spec_and_base_path(chain_spec, Default::default())
```

Comment:
I don't think that the default `config_dir` should be an empty path.

Reply:
If I see this correctly, this is only used in tests now, for which this doesn't matter. 

What would you suggest this should be used instead? Should I put in a tmp-path (though it is never used) or the current working directory? Or rename it to  ´default_with_spec_with_empty_path´ to make it explicit?

Reply:
Remove it?
If some tests need to call this, then it should call `Default::default()` itself and be aware that maybe something is written to this path.

Reply:
actually, no one seems to keep using it. removing.